### ISSUE TITLE
Cannot export TailwindComponent

### ISSUE THREAD

**Issue by taylor-ben** [MathiasGilson/Tailwind-Styled-Component#38]
Cannot export a component

```tsx
export const MyContainer = tw.div`flex`
```

Getting this error:
`Exported variable 'Container' has or is using name 'TailwindComponent' from external module ".../node_modules/tailwind-styled-components/dist/tailwind" but cannot be named.ts(4023)`

**Comment by taylor-ben** [comment]
Suggestion:

`./src/tailwind.tsx`
```tsx
export interface TailwindComponent
```

**Comment by Oldbury** [comment]
I'm getting this too, following for updates

**Comment by MathiasGilson** [comment]
Hello guys, 

I could not reproduce on my test repo.
Could you share your typescript version and your tsconfig.js ?

Many thanks

**Comment by MathiasGilson** [comment]
Okay I was able to reproduce, this was indeed due to not exporting the interface TailwindComponent.
It's now fixed in the version 2.1.1

thanks for reporting this issue!
